content/hi/docs/tasks/tools/_index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "उपकरण स्थापित करें"
+description: अपने कंप्यूटर पर कुबेरनेट्स टूल सेटअप करें.
+weight: 10
+no_list: true
+---
+
+## kubectl
+
+<!-- overview -->
+कुबेरनेट्स कमांड-लाइन टूल, [kubectl](/docs/reference/kubectl/kubectl/), 
+आपको कुबेरनेट्स क्लस्टर पर कमांड चलाने की अनुमति देता है।
+
+आप एप्लिकेशन को डिप्लॉय करने, क्लस्टर संसाधनों का निरीक्षण 
+और प्रबंधन करने और लॉग देखने के लिए kubectl का उपयोग कर सकते हैं।
+
+kubectl संचालन की पूरी सूची सहित अधिक जानकारी के लिए, देखें 
+[`kubectl` संदर्भ प्रलेखन](/docs/reference/kubectl/).
+
+kubectl विभिन्न प्रकार के Linux प्लेटफॉर्म, macOS और Windows पर इंस्टॉल करने योग्य है।
+नीचे अपना उपयुक्त ऑपरेटिंग सिस्टम खोजें।
+
+- [kubectl Linux पर इंस्टॉल करने](/docs/tasks/tools/install-kubectl-linux)
+- [kubectl macOS पर इंस्टॉल करने](/docs/tasks/tools/install-kubectl-macos)
+- [kubectl Windows पर इंस्टॉल करने](/docs/tasks/tools/install-kubectl-windows)
```

Comment:
If those pages also get localized, it would be good to go back and update the links to point to the Hindi pages.

Reply:
I think these are the file names so as we are not translating the file names to Hindi it's not an issue.

Reply:
@vedant-kakde I guess you misunderstood, we have PRs ready for these 3 files, so we can link to `hi` version of these.